CodeStarterCamp_Week3/main.swift

```diff
@@ -8,5 +8,37 @@
 
 import Foundation
 
-print("Hello, World!")
+class Person {
+    var money: Int = 0
+    
+    func buyCoffee() {
+        
+    }
+}
+
+class CoffeeShop {
+    var sales: Int = 0
+    var menu: [String: Int] = [:]
+    var pickUpTable: [String] = []
+    var barista: Person?
+    
+    enum Coffee: String {
+        case espresso, americano, cafeLatte, vanilaLatte, cafeMoca
+    }
+    
+    func getOrder(orderedCoffee: String) {
+        print("\(orderedCoffee)를 주문 받았다.")
+        let brewedCoffee: String = brewCoffee(to: orderedCoffee)
+        pickUpTable.append(brewedCoffee)
+    }
+    
+    func brewCoffee(to orderedCoffee: String) -> String {
```

Comment:
이 부분에도 Coffee 열거형 타입을 활용하면 좋겠네요 !!

일반적으로 더 친숙한 단어를 네이밍에 사용하면 좋다고 합니다. (brew 부분인데 굳이 바꾸지 않으셔도 괜찮습니다)
또한 최대한 간결하게 이름을 정해야 좋습니다. ☺️
전달인자 레이블을 활용하는 것도 좋지만, 중복된 의미를 갖는 단어를 사용한 네이밍을 피하기 위해 
이렇게 바꿔볼 수 있을 것 같아요 🤔
```suggestion
    func brew(coffee: Coffee) -> String {
```

Reply:
하나의 문장처럼 보이도록 한다는 것이 생각보다 어렵네요..😇
알려주셔서 감사합니다!